components/motor/gpio/motor_encoder.go

```diff
@@ -380,6 +387,11 @@ func (m *EncodedMotor) rpmMonitorPass(pos, lastPos, now, lastTime int64, rpmDebu
 					desiredRPM /= 2
 				}
 			}
+
+			if rpmDebug {
+				m.logger.Debugf("rot2go %.2f timeLef %.2f", rotationsLeft, timeLeftSeconds)
```

Comment:
I'm open to disagreements here, but I don't like the haphazard shorthand in these debugging statements, and would prefer full words. `Rotations remaining: %.2f, estimated time left: %.2f`

This comment applies in several other parts of the file, too.

Reply:
I actually prefer shorthand, especially since these messages are hidden under a layer of debugging that isn't available for a typical user. They will eventually move out of the code base so I prefer them as is.

Reply:
I think I agree with not using shorthand since we will have "users" that are writing their own motor drivers and those users will want to read these logs if they're using a custom motor with a Viam-supported encoder.